src/containers/guest-home-page/signup-dialog/SignupDialog.styles.js

```diff
@@ -11,7 +11,7 @@ export const styles = {
     maxHeight: { xs: 'calc(100vh - 56px)', sm: 'calc(100vh - 64px)' }
   },
   imgContainer: {
-    width: '100%',
+    width: '472px',
```

Comment:
Box size should always be calculated from content size.

Reply:
Fixed